apps/core/graphql/types/user.ex

```diff
@@ -0,0 +1,74 @@
+defmodule Core.GraphQL.Types.User do
+
+  use Absinthe.Schema.Notation
+  use Absinthe.Relay.Schema.Notation
+  alias Absinthe.Relay.Connection
+  alias Core.{GraphQL.Schema, User, Repo}
+  import Ecto.Query, only: [from: 2]
+  import Canada, only: [can?: 2]
+
+  #
+  # Mutations
+  #
+  object :user_mutations do
+    @desc """
+    Add/Edit a User to DB if success
+    """
+    payload field :user do
+      input do
+        field :id, non_null(:id)
```

Comment:
After we merge this pull request we're going to confirm the behavior of this and report a bug to Absinthe if necessary.